### PR TITLE
Add support for LaTeX format in output

### LINKED ISSUE #255: Support LaTeX output
Add the `--latex` option for the `dataize` command.

## Format

- Should we support a single format?
- Should we support the format used in the [phi-paper](https://github.com/yegor256/phi-paper/blob/750173428be0eedd5989ac3f153823737302fd60/sections/calculus.tex#L855-L865)?
  - Yes. this format is supported by the [eolang](https://ctan.org/pkg/eolang) LaTeX package ([link](https://github.com/objectionary/normalizer/issues/255#issuecomment-2073908758))

  image

## Example

Input program ([nop.phi](https://github.com/objectionary/normalizer/blob/a080ef0a49a91f93d3c901167387223f1322758c/eo-phi-normalizer/data/0.36.0/org/eolang/nop.phi)):

```
{
  ⟦
    org ↦ ⟦
      eolang ↦ ⟦
        nop ↦ ⟦
          args ↦ ∅,
          φ ↦ Φ.org.eolang.bool(
            α0 ↦ Φ.org.eolang.bytes(
              Δ ⤍ 01-
            )
          )
        ⟧,
        λ ⤍ Package
      ⟧,
      λ ⤍ Package
    ⟧
  ⟧
}
```

Output:

```
???
```

### PR DESCRIPTION
**Pull request by eyihluyc**
Closes #255

---

## PR-Codex overview
This PR updates the `eo-phi-normalizer` project to add LaTeX output support to the `transform` and `dataize` commands.

### Detailed summary
- Added `--tex` flag to output LaTeX in `transform` and `dataize` commands
- Introduced `Language.EO.Phi.ToLaTeX` module for LaTeX conversion
- Updated functions to support LaTeX output in `Main.hs`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: site/docs/src/normalizer/dataize.md [hunk 1]
```diff
@@ -65,7 +67,6 @@ normalizer dataize --chain --rules ./eo-phi-normalizer/test/eo/phi/rules/yegor.y
 
 ```console
 Dataizing inside phi: ξ.c.times (α0 ↦ ⟦ Δ ⤍ 02- ⟧).plus (α0 ↦ ⟦ Δ ⤍ 20- ⟧)
-Dataizing inside phi: ξ.c.times (α0 ↦ ⟦ Δ ⤍ 02- ⟧).plus (α0 ↦ ⟦ Δ ⤍ 20- ⟧)
```
Comment (deemp) [inline site/docs/src/normalizer/dataize.md:68]: I couldn't reproduce this change.

**Comment by deemp** [comment]
I made several suggestions in https://github.com/objectionary/normalizer/pull/314.

Additional suggestions:

- [ ] Use consistent naming: either `LaTeX` or `Latex`
- [ ] Update docs
  - [ ] Add a section about the `--latex` option. Provide examples there.

File: eo-phi-normalizer/app/Main.hs [hunk 2]
```diff
@@ -147,6 +150,9 @@ dependenciesArg =
 jsonSwitch :: Parser Bool
 jsonSwitch = switch (long "json" <> short 'j' <> help "Output JSON.")
 
+latexSwitch :: Parser Bool
+latexSwitch = switch (long "latex" <> short 'l' <> help "Output LaTeX.")
```
Comment (fizruk) [inline eo-phi-normalizer/app/Main.hs:154]: I think this one does not need a short name. And `--tex` is better than `--latex`:
```suggestion
latexSwitch = switch (long "tex" <> help "Output LaTeX.")
```

File: eo-phi-normalizer/app/Main.hs [hunk 3]
```diff
@@ -147,6 +150,9 @@ dependenciesArg =
 jsonSwitch :: Parser Bool
 jsonSwitch = switch (long "json" <> short 'j' <> help "Output JSON.")
```
Comment (fizruk) [inline eo-phi-normalizer/app/Main.hs:151]: By the way, this one probably also does not need a short option name.

File: site/docs/src/normalizer/dataize.md [hunk 4]
```diff
@@ -48,6 +49,7 @@ Available options:
                            output to stdout.
   --recursive              Apply dataization + normalization recursively.
   --chain                  Display all the intermediate steps.
+  -l,--latex               Output LaTeX.
```
Comment (fizruk) [inline site/docs/src/normalizer/dataize.md:52]: See comment above.
```suggestion
  --tex                    Output LaTeX.
```

File: site/docs/src/normalizer/transform.md [hunk 5]
```diff
@@ -71,7 +71,7 @@ normalizer transform --help
 
 ```console
 Usage: normalizer transform (-r|--rules FILE) [-c|--chain] [-j|--json]
-                            [-o|--output-file FILE] [-s|--single]
+                            [-l|--latex] [-o|--output-file FILE] [-s|--single]
```
Comment (fizruk) [inline site/docs/src/normalizer/transform.md:74]: See comment above.
```suggestion
                            [--tex] [-o|--output-file FILE] [-s|--single]
```

File: site/docs/src/normalizer/transform.md [hunk 6]
```diff
@@ -81,6 +81,7 @@ Available options:
   -r,--rules FILE          FILE with user-defined rules. Must be specified.
   -c,--chain               Output transformation steps.
   -j,--json                Output JSON.
+  -l,--latex               Output LaTeX.
```
Comment (fizruk) [inline site/docs/src/normalizer/transform.md:84]: See comment above.
```suggestion
  --tex.                   Output LaTeX.
```

**Comment by deemp** [comment]
`actions/checkout` merged `latex-option` into `master` ([link](https://github.com/objectionary/normalizer/actions/runs/8936724284/job/24549248619#step:2:173)). That's why we get a weird error about `MetaSubstThis` ([link](https://github.com/objectionary/normalizer/actions/runs/8936724284/job/24549248619#step:3:28685)) despite having no MetaSubstThis on `latex-option`. I suggest we merge `master` into `latex-option` and then fix the errors.

**Pull request merged**